app/shared-components/SteppedNavigation/SteppedNavigation.jsx

```diff
@@ -36,7 +36,7 @@ const SteppedNavigation = (props) => {
 	const circles = [];
 	for (let i = 1; i <= totalSteps; i++) {
 		// Use <a> without an href prop to prevent a call to history with the same URL
-		circles.push((i === activeIndex) ? <a className="active" key={`nav-${i}`} /> : <NavLink to={`${i}`} key={`nav-${i}`} />); // eslint-disable-line jsx-a11y/anchor-has-content
+		circles.push((i === activeIndex) ? <a className="active" key={`nav-${i}`} /> : <NavLink to={`${i}`} key={`nav-${i}`} />);
```

Comment:
There are two others linting rules like this.  On the video call with you, Ethan, Tino and me, Tino suggested that this could be something that interns could work on.
One of the other a11y linting rules will require strings and translations as it is for video captions.

Reply:
Right right. Sorry, I'd forgotten. Sounds good.